.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  mix_test:
+    name: mix test (Elixir ${{matrix.elixir}} | Erlang/OTP ${{matrix.otp}})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - elixir: 1.7.x
+            otp: 19.3.6.13
+          - elixir: 1.8.x
+            otp: 20.3.8.26
+          - elixir: 1.9.x
+            otp: 20.3.8.26
+          - elixir: 1.10.x
+            otp: 21.3.8.18
+          - elixir: 1.11.x
+            otp: 21.3.8.18
+          - elixir: 1.11.x
+            otp: 23.1.1
+            check_formatted: true
```

Comment:
Isn't warnings_as_errors: true missing here?

Reply:
Solved in 42a454dbef964105a6ba3cc3f291fdd638708453